### PR TITLE
add print style sheet

### PR DESCRIPTION
**Pull request by nunovieira**
Added a simple print style sheet

### CODE REVIEW

File: src/main/less/print.less [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+                    display: none;
+                }
+            }
+            ul.endpoints {
+                display: block !important;
+                li.endpoint {
+                    ul.operations {
+                        li.operation {
+                            div.content {
+                                display: block !important;
```
Comment (mohsen1) [inline src/main/less/print.less]: How about `li.endpoint ul.operations li.operation div.content {`?

File: src/main/less/print.less [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    display: none;
+}
+
+.swagger-section {
+
+.swagger-ui-wrap {
+
+    .model-signature {
+        pre {
+            max-height: none;
```
Comment (mohsen1) [inline src/main/less/print.less]: it's better to use `.model-signature pre {`

File: src/main/less/print.less [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+    input.parameter {
+        width: 100px;
+    }
+    
+    ul#resources {
+        li.resource {
+            div.heading {
+                ul.options {
+                    display: none;
```
Comment (mohsen1) [inline src/main/less/print.less]: `ul#resources li.resource div.heading ul.options {`

**Comment by mohsen1** [comment]
The file `src/main/html/css/print.css` is extra. There is no need to have it. Please remove it and fix those nesting issue.

Please fix those issues and make the PR to `develop_2.0` branch.

**Comment by nunovieira** [comment]
The file `src/main/html/css/print.css` is generated automatically by gulp. Don't think I should remove it, since it will be recreated on every gulp rebuild.

I can't change this PR branches. Should I create a new PR?

**Comment by mohsen1** [comment]
Yes, please make a new PR. Don't worry about `src/main/html/css/print.css` for now

**Comment by nunovieira** [comment]
Created PR #973, as requested

**Pull request closed without merging**